src/components/TimecodeTracker/TimecodeInput.js

```diff
@@ -29,19 +29,20 @@ export default function TimecodeInput({ title, index, inputValue, onChange }) {
 }
 
 const InputContainer = styled.div`
-  display: flex;
-  flex-direction: column;
   align-items: start;
-  border: 1px solid #ddd;
   border-radius: 5px;
+  border: 1px solid #ddd;
+  display: flex;
+  flex-direction: column;
+  margin: 10px 0;
 `
 
 const Label = styled.label`
-  display: inline-block;
-  margin-bottom: 2px;
   color: #737373;
-  font-size: 18px;
+  display: inline-block;
   font-size: 150%;
+  font-size: 18px;
```

Comment:
You should consider writing font-sizes in em or rem instead of px. This would make the app more accessible for people with custom font-sizes in their browser.

Reply:
will do that, when working on design